_posts/2018-03-16-first-steps-into-gen-servers.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Automating Processes with Gen Servers: A Beginner's Approach"
+date:   2018-03-16 10:00:00 -0400
+categories: elixir automation microservices
+author: Kelvin Ma
+---
+As a financial services company, Forward Financing invests a signficant amount of time and resources around timebound reporting -- think daily, weekly, monthly, etc., report generation. As opportunities for automation are concerned, it doesn't get more obvious than this.
+
+A big part of my growth as an engineer comes from getting out of my comfort zone, using new and unfamiliar tools to build hopefully useful tools. After doing a little research on potential solutions for our automation opportunity, I settled on Elixir's GenServer, or Generic Server, pattern. Coming from a primarily Object Oriented Programming background with Rails, this pattern was a bit foreign to me, to say the least.
```

Comment:
The first sentence of this paragraph is a bit awkward. I'd drop 'hopefully' and only use 'tools' once. Maybe 'using new and unfamiliar technologies to build useful tools'?